.stylelintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "tailwind",
+          "apply",
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ]
+  },
+  "csstree/validator": true,
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+}

```

Comment:
- [ ] As previous reviewer didn't mentioned, it is highly suggested to add this linter structure to your project. Right now, I can't see the green tick ✅ on your linters. Add this github/workflows folder into your repository to have it. 
![image](https://user-images.githubusercontent.com/91486791/199067625-b9e8c415-b40e-458a-9b09-7448b4225bbe.png)
![image](https://user-images.githubusercontent.com/91486791/199067815-35200d39-85d7-42d5-b307-20aa0c474ec1.png)